lib/plugins/entry-files-manifest.js

```diff
@@ -9,26 +9,41 @@
 
 'use strict';
 
-const EntryFilesManifestPlugin = require('../webpack/entry-files-manifest-plugin');
 const PluginPriorities = require('./plugin-priorities');
-const path = require('path');
-const sharedEntryTmpName = require('../utils/sharedEntryTmpName');
-const manifestKeyPrefixHelper = require('../utils/manifest-key-prefix-helper');
```

Comment:
It seems a bit strange to remove that require (and therefore not use that helper anymore).
Does it mean that the keys won't be prefixed the same way than before?

Reply:
They come out already prefixed based on the webpack settings. I will dig more into seeing if this plugin that was picked out will allow us to pass in another prefix.

Reply:
This comment makes it look like I removed sharedEntryTmpName. I have put it back after realizing I had removed a module that should not have been removed and it is in the current commit.